lms/djangoapps/organizations/management/commands/tests/test_move_organizations_entries.py

```diff
@@ -40,7 +41,7 @@ def setUp(self):
             org.groups.add(group1, group2)
             org.workgroups.add(workgroup)
 
-
+    @skip
```

Comment:
Yeah this test is no longer applicable. It is related to moving data from lms/api_manager/organizations to lms/organizations app. Removing this.